lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -1,26 +1,26 @@
-# (C) British Crown Copyright 2013 - 2016, Met Office
+# (C) British Crown Copyright 2013 - 2017, Met Office
 #
-# This file is part of Iris.
+# This file is part of iris.
 #
-# Iris is free software: you can redistribute it and/or modify it under
+# iris is free software: you can redistribute it and/or modify it under
 # the terms of the GNU Lesser General Public License as published by the
 # Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
 #
-# Iris is distributed in the hope that it will be useful,
+# iris is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
-# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+# along with iris.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
These should be `Iris` with a capital `I` to be consistent with the other files.